tests/disas/pulley/memory-inbounds.wat

```diff
@@ -0,0 +1,31 @@
+;;! target = "pulley64"
+;;! test = "compile"
+
+(module
+  (memory 1)
```

Comment:
Would it make sense to specify a max, like`(memory 1 2)`, so that this can test for loads that are in-bounds on the minimum but not the maximum?

Reply:
> so that this can test for loads that are in-bounds on the minimum but not the maximum?

I think this should be "in bounds of the maximum but not the minimum", but yeah +1